File: Python/Program/HDD.py
from Komponen import Komponen

class HDD(Komponen):
    def __init__(self, kapasitas_gb, tipe_drive, merk, nama, jenis_komponen):
        super().__init__(merk, nama, jenis_komponen)
        self.kapasitas_gb = kapasitas_gb
        self.tipe_drive = tipe_drive

    def get_kapasitas_gb(self):
        return self.kapasitas_gb

    def get_tipe_drive(self):
        return self.tipe_drive

File: C++/Program/HDD.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class HDD : public Komponen
{
private:
    int kapasitasGB;

public:
    HDD()
    {
    }

    HDD(int kapasitasGB, string merk, string nama, string jenis_komponen) : Komponen(merk, nama, jenis_komponen)
    {
        this->kapasitasGB = kapasitasGB;
    }

    void setKapasitasGB(int kapasitasGB)
    {
        this->kapasitasGB = kapasitasGB;
    }

    int getKapasitasGB()
    {
        return this->kapasitasGB;
    }

    ~HDD()
    {
    }
};